types/web-bluetooth/tsconfig.json

```diff
@@ -3,6 +3,7 @@
         "module": "commonjs",
         "lib": [
             "es6",
+            "ESNext",
```

Comment:
Why is this needed?

Reply:
This is down to .padStart being done in the test. I could probably just rewrite it, or specify it better as ES2017.String? It's not needed for the types themselves however
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart

Reply:
You should be able to get a similar effect with something along the lines of:

```typescript
`0${b.toString(16)}`.slice(-2);
```

Reply:
Makes sense, I've changed it now. That being said, I'm a little curious, seeing as Web Bluetooth will anyway only be available in browsers that supports the ES2017.String spec, would there be an argument to considering those things in the future?

Thanks for the review though, much appreciated 😄

Reply:
Thanks for the changes, it's just always good to avoid extra prereqs/dependencies wherever possible IMO